Base/Logic/Resources/ColorFiles/Terminology/GenericAnatomyColors-SNOMED.csv

```diff
@@ -0,0 +1,291 @@
+# Integer Label,Text Label,Segmented Property Category -- CID 7150++,Segmented Property Type,Segmented Property Type Modifier,Color
+# Version 2
+SlicerLUT=GenericAnatomyColors
+1,tissue,(T-D0050;SRT;Tissue),(T-D0050;SRT;Tissue),,rgb(128;174;128)
```

Comment:
Yet another idea - maybe we should just add this statement to the comment section of the .csv file?